docs/cudos-intercloud/usage/via-user-interface/profile.md

```diff
@@ -32,3 +37,8 @@ An additional identity has been successfully added to your CUDOS Intercloud acco
 Keplr and MetaMask are currently supported. Cosmostation and more will follow shortly.
```

Comment:
I would stop specifically promising Cosmostation, shortly. I don't think it's very high up the priority list.